bblfsh/client.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+
+import grpc
+
+# The following two insertions fix the broken pb import paths
+sys.path.insert(0, os.path.join(os.path.dirname(__file__),
+                                "github/com/bblfsh/sdk/protocol"))
+sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2 import ParseUASTRequest
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2_grpc import ProtocolServiceStub
+
+
+class BblfshClient(object):
```

Comment:
There is no close(). Python does it automatically without ResourceWarning-s. Even the Python grpc primer does not close anything: http://www.grpc.io/docs/tutorials/basic/python.html